app/src/test/java/com/trueedu/project/model/dto/WsResponseParseTest.kt

```diff
@@ -15,4 +15,17 @@ class WsResponseParseTest {
         val wsResponse = json.decodeFromString<WsResponse>(jsonString)
         assertEquals(TransactionId.PingPong, wsResponse.header.transactionId)
     }
+
+    @Test
+    fun `실시간 가격 요청 성공에 대한 응답 처리`() {
+        val json = Json { ignoreUnknownKeys = true }
+
+        val jsonString = "{\"header\":{\"tr_id\":\"H0STASP0\",\"tr_key\":\"066570\",\"encrypt\":\"N\"},\"body\":{\"rt_cd\":\"0\",\"msg_cd\":\"OPSP0000\",\"msg1\":\"SUBSCRIBE SUCCESS\",\"output\":{\"iv\":\"ca513e8bf725c575\",\"key\":\"uvbpznjqabrxhowcftgehxghhhsxtwro\"}}}"
```

Comment:
https://kotlinlang.org/docs/strings.html#multiline-strings

```suggestion
        val jsonString = """{"header":{"tr_id":"H0STASP0","tr_key":"066570","encrypt":"N"},"body":{"rt_cd":"0","msg_cd":"OPSP0000","msg1":"SUBSCRIBE SUCCESS","output":{"iv":"ca513e8bf725c575","key":"uvbpznjqabrxhowcftgehxghhhsxtwro"}}}"""
```

Reply:
ㄱㅅ합니다